Actions/Deliver/Deliver.ps1

```diff
@@ -77,7 +77,7 @@ try {
         }
         # projectName is the project name stripped for special characters
         $projectName = $project -replace "[^a-z0-9]", "-"
-        Write-Host "Project '$project'"
+        Write-Host "ProjectName '$projectName'"
```

Comment:
```suggestion
        Write-Host "Project name: '$projectName'"
```

Reply:
It is not the Project Name - it is dumping the value of ProjectName for troubleshooting scenarios.